src/App.js

```diff
@@ -434,10 +426,16 @@ export default class App extends Base {
           }
 
           // Set the pubSub URL if present
+          console.log('2507 test log - query params', query.params);
           if (query.params.pubSubUrl) {
             process.env.PUB_SUB_URL = query.params.pubSubUrl;
+            console.log('2507 test log - query params pubsuburl', process.env.PUB_SUB_URL);
+          }
+          // Set the pubSub token if present
+          if (query.params.pubSubToken) {
```

Comment:
We should add an entry to search for this in the URL as well

Reply:
addressed